src/vaadin-combo-box-mixin.html

```diff
@@ -742,7 +754,7 @@
     }
 
     _itemsOrPathsChanged(e, itemValuePath, itemLabelPath) {
-      if (e.value === undefined || itemValuePath === undefined || itemLabelPath === undefined) {
```

Comment:
Same as above: please revert the change to this line in case if new logic does not require it.

Reply:
Discussed in person: new test is failing without this change. OK to include it.